lib/features/auto-resize/BpmnAutoResizeProvider.js

```diff
@@ -33,6 +33,12 @@ BpmnAutoResizeProvider.$inject = [
  */
 BpmnAutoResizeProvider.prototype.canResize = function(elements, target) {
 
+  // do not resize plane elements:
+  // root elements, collapsed sub-processes
+  if (is(target.di, 'bpmndi:BPMNPlane')) {
+    return false;
```

Comment:
We do have test to verify this, correct? I.e. creating or moving elements on a root element?